content/user/installing/pick_hosting.md

```diff
@@ -12,5 +12,12 @@ As noted in [Basics - Hosting and Domain concepts](/user/first_steps/hosting/#ho
 their [Recommended Services](https://www.zen-cart.com/content.php?3-services) page.
 
 Further Considerations: 
-- One way to evalute hosts is to run the [test suite offered SSLLabs](https://ssllabs.com/ssltest).  You will want to make sure any host you consider has at least a B rating.  This will help make any PCI scan more likely to succeed.  The more secure your host, the more secure your site.
+
+- One way to evaluate hosts is to run the [test suite offered by SSLLabs](https://ssllabs.com/ssltest).  You will want to make sure any host you consider has an A or A+ rating.  (A B rating might be ok if it's something that can be easily fixed by them, but exercise caution.)  
+
+   A highly rated host will help make any PCI scan you run more likely to succeed.  The more secure your host, the more secure your site.
+
+   Don't be like these guys! 
+
+![SSL Test - C Grade](/images/ssltest_c_grade.png)
```

Comment:
Rather than "don't be like", perhaps a more ideal tone (more suitable for our docs) would be a positive example?

Reply:
Fair - but my intent was to show a failure case as a warning.  I could show A, B and C results.

Reply:
The phrase "Don't be like these guys!" doesn't sit well with me.

Reply:
Agreed - how about "Here's an example of a site with a poor grade"

Reply:
That'd be fine if the image shows examples of an A and C grade.

Reply:
will do.  Just removed the objectionable wording a minute ago.